components/activity.js

```diff
@@ -1,38 +1,42 @@
-import { React } from 'react';
+import { React } from "react"
 
-import {
-  Header,
-  Icon,
-  Container
-} from 'semantic-ui-react';
-import Counter from './counter.js';
-import Players from './playerlist.js';
+import { Header, Icon, Container } from "semantic-ui-react"
+import Counter from "./counter.js"
+import Players from "./playerlist.js"
 
-import { server } from '../data.js';
+import { server } from "../data.js"
 
 function _Activity() {
-  const {data, isLoading, isError} = server();
+  const { data, isLoading, isError } = server()
 
-  if (isError) return (
-    <Header size="huge" style={{color:"red"}}>
-      Server seems down.
-      <br/>
-      Please contact
-      <span style={{color:"blue"}}> <Icon name="discord" fitted/> Fl_GUI#5136 </span>
-      on discord.
-    </Header>
-  );
+  if (isError)
+    return (
+      <Header size="huge" style={{ color: "red" }}>
+        Server seems down.
+        <br />
+        Please contact
+        <span style={{ color: "blue" }}>
+          {" "}
```

Comment:
There are a bunch of `{" "}` scattered around because of the linter. I could easily remove these automatically with vim if you want.

Reply:
I think some of them are necessary because words are on the line after, but there are definitely cases like this. I'll take care of them, but thanks 👍